build.gradle

```
@@ -0,0 +1,38 @@
+plugins {
+    id 'org.springframework.boot' version '3.1.3'
```

```suggestion
    id 'org.springframework.boot' version '3.1.3' apply false
```
여기에 플러그인을 적용하면, 루트 디렉토리도 스프링 부트 소스로 인식하게 되서 그걸 방지했으면 좋겠습니다.